### DISCUSSION TITLE
Podcast - Saving the SpaceOrb360 w/open source hardware, software with Vic Putz . Great podcast! Highly informative

### DISCUSSION THREAD

**Discussion by gamer85** [thingotron/orb9k_circuitpython#2] [General]
Podcast - Saving the SpaceOrb360 w/open source hardware, software with Vic Putz . *Great podcast! Highly informative*

https://hanselminutes.simplecast.com/episodes/35e5a63d-35e5a63d
![poster](https://user-images.githubusercontent.com/20948864/172496381-c7f17107-c57f-4a32-8b1e-087319fc46e3.jpg)

**Comment by petersem** [comment]
Excellent podcast
